pyproject.toml

```diff
@@ -0,0 +1,94 @@
+[tool.poetry]
+name = "bwapy"
+version = "0.0.1-dev"
+description = "Python bindings for BWA"
+authors = ["Nils Homer"]
+license = "MIT"
+readme = "README.md"
+homepage = "https://github.com/fulcrumgenomics/bwapy"
+repository = "https://github.com/fulcrumgenomics/bwapy"
+keywords = ["bioinformatics"]
+packages = [{ include = "bwapy" }]
+classifiers = [
+	"Development Status :: 3 - Alpha",
+	"Environment :: Console",
+	"Intended Audience :: Developers",
+	"Intended Audience :: Science/Research",
+	"License :: OSI Approved :: MIT License",
+	"Operating System :: OS Independent",
+	"Programming Language :: Python :: 3",
+	"Topic :: Scientific/Engineering :: Bio-Informatics",
+	"Topic :: Software Development :: Documentation",
+	"Topic :: Software Development :: Libraries :: Python Modules",
+]
+include = [
+	"LICENSE",
+	{ path = "bwapy/**/*.so", format = "wheel" },
+	{ path = "**/*.so", format = "wheel" },
+	{ path = "*.so", format = "wheel" },
+	#{ path = "*.so", format = "wheel" }
+]
+
+[tool.poetry.build]
+generate-setup-file = false
+script = "build.py"
+
+[tool.poetry.dependencies]
+python = ">=3.9.0,<4.0"
+pysam = ">=0.22.1"
+typing_extensions = { version = ">=3.7.4", python = "<3.12" }
+
+[tool.poetry.group.dev.dependencies]
+# dependencies for linting, style checking, and unit testing
+mypy = ">=1.7.0"
+pytest = ">=7.4.0"
+pytest-cov = ">=2.8.1"
+ruff = "0.4.8"
+setuptools = ">=68.0.0"
```

Comment:
We typically use `tim-fmt` and align assignment operators within each TOML section.

Reply:
Perhaps this could be an opportunity to break that pattern.

Reply:
💯  

I would welcome that change in policy. 

I'm always in favor of avoiding stylistic requirements that can't be enforced with an automated tool. (No matter your flavor of code review pyramid, I think style is always at the top. 🙂 )

However, since I have had PRs blocked over this particular requirement, I think it'd be helpful if the decision to relax it were communicated more broadly, so contributors and reviewers are aware it should no longer be blocking.

Reply:
Is this a policy written down somewhere or an emergent policy?